packages/terra-dropdown-button/src/Button.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Should this file be Option.jsx? Or should the class here be renamed to Button?

Reply:
It used to be `Option.jsx` but it was decided it should be renamed from `Option` to `Button`. Guess the function name didn't get changed.

Reply:
Fixed in fa6e738959360e067e0c7ca807365bedf22725a8